### DISCUSSION TITLE
请问主题右侧日历栏下方怎么添加小组件

### DISCUSSION THREAD

**Discussion by scgmwwx** [Tokinx/Wing#46] [问答]
[https://wonderfulworldstudio-1304739211.cos.ap-nanjing.myqcloud.com/%E6%96%87%E7%AB%A0%E7%94%A8%E5%9B%BE/2022%E5%B9%B4/6%E6%9C%88/20220911125020.png](url)
如图所示

**Comment by Tokinx** [comment]
图片挂掉了。。。

**Comment by moenyi** [comment]
发布一个随笔，#一个话题，那边就出现 了

**Reply by scgmwwx** [reply]
感谢！明白了
